### PR TITLE
clean up move_relearner.c

### PR DESCRIPTION
**Pull request by red031000**
WIP

### CODE REVIEW

**Comment by red031000** [comment]
build should work now

**Comment by red031000** [comment]
seeking opinions on the renaming of wotbl, I suggest lvlUpTable

**Comment by adrienntindall** [comment]
> seeking opinions on the renaming of wotbl, I suggest lvlUpTable

learnSetTable, moveLearnTable, levelUpTable, etc

**Comment by red031000** [comment]
ready for review

File: src/move_relearner.c [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    memset(ret, 0, sizeof(MoveRelearner));
+    return ret;
+}
+
+void MoveRelearner_Delete(MoveRelearner *moveRelearner) {
+    FreeToHeap(moveRelearner);
+}
+
+u16 *MoveRelearner_GetEligibleLevelUpMoves(Pokemon *mon, HeapID heapId) {
+    u16 species = (u16)GetMonData(mon, MON_DATA_SPECIES, NULL);
```
Comment (adrienntindall) [inline src/move_relearner.c:16]: casts here are unnecessary

File: src/pokemon.c [hunk 2]
```diff
@@ -3042,26 +3042,26 @@ u16 GetEggSpecies(u16 species) {
...
     BOOL decry;
-    u16 * wotbl;
+    u16 *levelUpLearnset;
     int i;
     u16 species;
     u32 form;
     u8 level;
     u16 move;
-    wotbl = AllocFromHeap(HEAP_ID_DEFAULT, 22 * sizeof(u16));
+    levelUpLearnset = AllocFromHeap(HEAP_ID_DEFAULT, 22 * sizeof(u16));
```
Comment (adrienntindall) [inline src/pokemon.c:3051]: const for LEARNED_MOVES_MAX = 22?

**Review by adrienntindall — CHANGES_REQUESTED** [review]

**Review by adrienntindall — APPROVED** [review]

**Pull request merged**